mathics/builtin/box/compilation.py

```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8 -*-
 
 from mathics.builtin.base import BoxConstruct
+from mathics.core.symbols import ExpandOnce
 
 
-class CompiledCodeBox(BoxConstruct):
+class CompiledCodeBox(BoxConstruct, ExpandOnce):
```

Comment:
I don't understand why, but ExpandOnce has to be included here, even though it is mentioned in BoxConstruct. I googled and see one might try `issubclass()` instead of `isinstance()`, but I actually see a benefit in listing it ere even if it were unnecessary so as to make the intent explicit.

Reply:
Again, because of the hackiness needed to check it, and since `ExpandOnce` does not define a different interface, shouldn't be simpler to add a property to the class?